suite-native/module-dev-utils/src/components/FeatureFlags.tsx

```diff
@@ -0,0 +1,23 @@
+import { Box, CheckBox, Text, VStack, Card } from '@suite-native/atoms';
+import { useIsDeviceConnectEnabled } from '@suite-native/feature-flags';
+
+export const FeatureFlags = () => {
+    const { isDeviceConnectEnabled, setIsDeviceConnectEnabled } = useIsDeviceConnectEnabled();
+
+    return (
+        <Card>
+            <VStack spacing="small">
+                <Text variant="titleSmall">Feature Flags</Text>
+                <VStack>
+                    <Box flexDirection="row" justifyContent="space-between">
+                        <Text>Connect device</Text>
+                        <CheckBox
```

Comment:
Would be awesome if there would be a checkbox for every flag that we define in `suite-native/feature-flags` wihtout explicitly naming them here. But this is not related to this issue, could be maybe done in a follow up issue. Just thinking out loud.

Reply:
Yeah, when we have more FFs, we'll do it like this for sure, now it's just a kick off and don't worry about it in this PR